spack/configs/summit/compilers.yaml

```diff
@@ -17,13 +17,25 @@ compilers:
     spec: gcc@8.1.1
     modules: [gcc/8.1.1]
     paths:
-      cc: /sw/summit/gcc/8.1.1/bin/gcc
-      cxx: /sw/summit/gcc/8.1.1/bin/g++
-      f77: /sw/summit/gcc/8.1.1/bin/gfortran
-      fc: /sw/summit/gcc/8.1.1/bin/gfortran
+      cc: gcc
+      cxx: g++
+      f77: gfortran
+      fc: gfortran
```

Comment:
In the example in the docs (https://spack.readthedocs.io/en/latest/getting_started.html#compilers-requiring-modules), they still use the full path to the compilers even when using modules. It probably doesn't really make a difference, but I think I'd prefer to having the full path since it seems safer.

Reply:
Thanks for the link.  After reading the docs, I agree that going with the full path is safer; it says the module is loaded only when running the compiler... but, as you suggest, it seems like there could be some other !running step that could benefit from the full path.

Reply:
I just forced pushed the change to use full paths.